launchd/smoketestbuild.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e -u -o pipefail # Fail on error
+
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd "$dir"
+
+logpath=${LOG_PATH:-}
+
+client_dir="$GOPATH/src/github.com/keybase/client"
+
+echo "Loading release tool"
+"$client_dir/packaging/goinstall.sh" "github.com/keybase/release"
+release_bin="$GOPATH/bin/release"
+
+err_report() {
+  url=`$release_bin save-log --bucket-name=prerelease.keybase.io --path=$logpath --noerr`
+  "$client_dir/packaging/slack/send.sh" "Error see $url"
+}
+
+trap 'err_report $LINENO' ERR
+
+if [ -n "$SMOKETEST_BUILD_A" ] && [ -n "$SMOKETEST_BUILD_PLATFORM" ] && [ -n "$SMOKETEST_BUILD_ENABLE" ]; then
```

Comment:
I wonder if we could use SCRIPT_PATH and run.sh? and this script would only be this small part, that way we don't have to dupe run.sh code?